### DISCUSSION TITLE
Water tank sensor question

### DISCUSSION THREAD

**Discussion by big-col** [hatlabs/discussions#112] [General]
Hi, i have a 4 stage water tank sensor and gauge. By 4 stage, i mean 1/4, 1/2, 3/4, 1/1 (full).  pics attached.
Am i right in thinking these would need to be connected as individual digital connections, unlike a variable resistive type
![watertankgauge](https://github.com/hatlabs/discussions/assets/10856730/eebc3daa-d12f-45c1-adf9-b16aa55ffa85)
![watertankprobe](https://github.com/hatlabs/discussions/assets/10856730/37810831-b486-453d-9d3d-6b2378bf0267)

Actually i have 2 of these on separate water tanks

**Comment by mairas** [comment]
If you'd patch the device into the LED outputs of the control panel, then yes. Depending on how the control panel works, using the GPIO pins for that purpose might work just fine. But if the device is super simple, it's possible that the LEDs are driven straight from the 12V input and that would make things a bit more complex.

Then again, just changing the sensor to a resistive one could be the path of, ahem, least resistance...

**Reply by big-col** [reply]
😂😂😂😂.  A new sensor may well be ordered before the end of the day...

**Comment by big-col** [comment]
Thank for a quick response.

**Comment by mairas** [comment]
Re-opening to provide visibility for future readers.
